src/humanloop/evals/run.py

```diff
@@ -467,10 +392,11 @@ def _wait_for_evaluation_to_complete(
             if not waiting_for_local_evals_message_printed:
                 sys.stderr.write("\n\nWaiting for Evaluators on Humanloop runtime...\n")
                 waiting_for_local_evals_message_printed = True
-            sys.stderr.write(stats.progress)
+            sys.stderr.write(stats.progress)  # type: ignore [arg-type]
             # Move the cursor up in stderr a number of lines equal to the number of lines in stats.progress
-            sys.stderr.write("\033[A" * (stats.progress.count("\n")))
+            # sys.stderr.write("\033[A" * (stats.progress.count("\n")))
             time.sleep(5)
```

Comment:
fix/followup: we did say to move away from this polling behavior. Prefer separate PR for it though